android_sample/build.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+if [[ $ANDROID_SDK_HOME == "" ]]; then
+  echo "Error: set ANDROID_SDK_HOME environment variable and run it again e.g.,"
+  echo "       $ export ANDROID_SDK_HOME=path/to/Android/SDK"
+  exit 1
+fi
+
+if [[ $ANDROID_NDK_HOME == "" ]]; then
+  echo "Error: set ANDROID_NDK_HOME environment variable and run it again e.g.,"
+  echo "       $ export ANDROID_NDK_HOME=path/to/Android/NDK"
+  exit 1
+fi
+
+APK_NAME=Amber.apk
+ANDROID_PLATFORM=android-28
+ANDROID_BUILD_TOOL_VERSION=28.0.0
+ABI=arm64-v8a
+BUILD_TYPE=Debug
+
+AAPT=$ANDROID_SDK_HOME/build-tools/$ANDROID_BUILD_TOOL_VERSION/aapt
+AAPT_ADD="$AAPT add"
+AAPT_PACK="$AAPT package -f -I
+           $ANDROID_SDK_HOME/platforms/$ANDROID_PLATFORM/android.jar"
+
+DX="$ANDROID_SDK_HOME/build-tools/$ANDROID_BUILD_TOOL_VERSION/dx --dex"
+
+JAVAC="javac -classpath
+       $ANDROID_SDK_HOME/platforms/$ANDROID_PLATFORM/android.jar"
+JAVAC_BUILD="$JAVAC -sourcepath build/gen -d build"
+
+ANDROID_CMAKE=`find $ANDROID_SDK_HOME -name 'cmake' -executable -type f`
+ANDROID_NINJA=`find $ANDROID_SDK_HOME -name 'ninja' -executable -type f`
+
+KEYTOOL_PASSWORD=changeit
+
+rm -rf build/$APK_NAME classes.dex com/ gen/
+
+mkdir -p build/gen build/output/lib/$ABI build/$BUILD_TYPE
+
+echo "Step 1:"
```

Comment:
Remove the echo lines and add:

set -x

at the top of the script to have bash do the echo automatically.